iina/Base.lproj/Localizable.strings

```diff
@@ -167,6 +167,12 @@
 "preference.sub_override_level.strip" = "strip";
 "preference.sub_override_level.yes" = "yes";
 
+"preference.sub_override_level.descriptive_text.no" = "No override: render subtitles as specified by the subtitle scripts.";
+"preference.sub_override_level.descriptive_text.force" = "Only allow override using the --sub-ass-* mpv options.";
+"preference.sub_override_level.descriptive_text.scale" = "Allow adjusting scale of ASS subtitles and override using the --sub-ass-* mpv options.";
+"preference.sub_override_level.descriptive_text.strip" = "Allow adjusting scale and style of ASS subtitles.";
```

Comment:
This is not what `strip` does. For `strip` I would use the mpv manual text:
`Radically strip all ASS tags and styles from the subtitle.`